### DISCUSSION TITLE
How to getcurrent specname along with scenario /tags to place screenshots to corresponding scenarios/

### DISCUSSION THREAD

**Discussion by prem1321** [getgauge/gauge#2076] [General]
How to getcurrent specname along with scenario /tags to place screenshots to corresponding scenario folders.

**Comment by sriv** [comment]
You can get this in any hooks (ex. `BeforeScenario`) using the [`ExecutionContext`](https://javadoc.io/doc/com.thoughtworks.gauge/gauge-java/latest/com/thoughtworks/gauge/ExecutionContext.html) passed in.
